section_start_rules.json

```diff
@@ -528,23 +528,53 @@
         }
     },
     "Section-Excerpt-Opener (seo)": {
-        "section_type": "backmatter"
-    },
-    "Section-Excerpt-Chapter (sec)": {
         "section_type": "backmatter",
+        "order": "first",        
         "contiguous_block_criteria_01": {
-            "multiple": true,
+            "multiple": false,
```

Comment:
Remind me: does `"multiple":false` mean we do not need to search for multiple occurrences, or multiple are not allowed? If the former, then this is OK.

Reply:
Sort of the same outcome..  it's not removing anything, but it won't add a section start for a second occurrence.  This is a little confusing here, because it applies to the "Section-Excerpt-Opener (seo)", not the "Section-Excerpt-Chapter (sec)"... though I guess there may be occasions where you have a second SEO? let me know what you think & I will update if necessary:)

Reply:
There could be situations where a book has two excerpts (I think books in a series sometimes get an excerpt from the next in the series and the most recent in the series), so I think this should be `"multiple":true`, as long as no other changes would be required for that.